docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -4,50 +4,251 @@
+
+.. image:: causes_equation.svg
+
+All-cause mortality is sum of all cause-specific mortalities: 
+
+   ACMR =  :math:`\sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{CSMR}`
+
+The mortality from unmodelled causes affected by LBWSG (blue) is thus: 
+
+   :math:`\sum\limits_{\text{blue}}\text{CSMR}` = ACMR - :math:`\sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR}`
+
+Because we model some of the causes affected by LBWSG, we can use their excess-morality rates (EMR) instead of the average CSMRs: The mortality from modelled causes affected by LBWSG (green):
+
+   - cause-specific mortality if the person who does NOT have the condition: 0
+   - cause-specific mortality if the person HAS the condition: EMR of the condition
+
+ 
+We are interested in applying the PAF and relative risk to only the causes that GBD considers to be affected by LBWSG (green and blue):
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in early and late neonatal period for category i
+   |  rr_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+   |  PAF* = this is the PAF of the most detailed cause affected by LBWSG
+
+Hence, the mortality hazard for an individual in LBWSG category i is:  
+
+mr_i 
+
+   | = ACMR_i 
+   | = (sum of unaffected causes) + affected(sum of unmodelled + sum of modelled) x (1-PAF*) x :math:`rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (\sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (ACMR - \sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+
+.. important :: 
+
+   PAF in the above equation represents PAFs for most-detailed-cause (they are all roughly equal) affected by LBWSG (or as calculated in vivarium from LBWSG relative risks and exposure). This approach assumes that relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG (green and blue causes). 
+
+   `LBWSG PAF notebook <https://github.com/ihmeuw/vivarium_data_analysis/blob/master/pre_processing/lbwsg/LBWSG%20exposure%2C%20rrs%2C%20pafs.ipynb>`__.
+.. todo::
+   - add more description of the all-causes PAF and mode-detailed-cause PAF and the logical reasoning for using one over the other. 
+   - here, we can also discuss the other equations that thought of but did not end up using. 
+
+   
+Assumptions and limitations
++++++++++++++++++++++++++++
+ 
+Strengths
+
+   o  This approach is consistent with GBD methodology and avoids artificially decreasing the mortality rate for individual causes that are not affected by improvements in LBWSG (due to reverse causality or other concerns).
+   
+Limitations
```

Comment:
I think Ali mentioned a separate limitation yesterday, which was that GBD assumed that the RR's for CSMR for each affected cause were the same as the overall RR for ACMR. This won't matter for the blue causes that we aren't modeling explicitly, but for the green causes that we _are_ modeling, it could throw off our results depending on whether the RR's for that cause is larger or smaller than the overall RR for all causes.